README.md

```diff
@@ -2,197 +2,230 @@
 
 [![Build Status](https://travis-ci.org/vtex/toolbelt.svg?branch=master)](https://travis-ci.org/vtex/toolbelt) [![npm version](https://img.shields.io/npm/v/vtex.svg?style=flat)](https://www.npmjs.com/package/vtex) [![Greenkeeper badge](https://badges.greenkeeper.io/vtex/toolbelt.svg)](https://greenkeeper.io/)
 
-All you need to start using the VTEX platform.
+The CLI that offers all you need to start using the VTEX IO platform efficiently.
+
+#### Summary
+- [How do we empower you?](#how-do-we-empower-you)
+- [First step: Installing the VTEX Toolbelt](#installing-the-toolbelt-with-yarn)
+- [Developing an app locally](#to-develop-locally-use-vtex-link)
+- [Customize your prompt](#wanna-know-something-cool-you-can-customize-your-prompt)
+- [FAQ](#frequently-asked-questions)
+- [Contributing](#contributing)
+- [Send us feedback](#tell-us-what-you-think-about-toolbelt)
+
+# How do we empower you? 
+
+VTEX Toolbelt is a command-line program that gives you access to all of 
+the features in the VTEX platform and increases your productivity, allowing you to:
+
+- **Manage your apps quickly:** List, install and uninstall available apps in your account;
+- **Work safely on your personal environments:** List, create and remove workspaces;
+- **Easily ship changes to your final user:** Promote a workspace to `master`, making it public in your account URL;
+- **See what you are creating in real time:** Develop new apps in link mode, using automatic synchronization;
+- **Go live from the command line:** Publish apps to the VTEX App Store.
+
+# Getting started: Installing the VTEX Toolbelt
+To use the VTEX Toolbelt, you'll need to install other engines 
+(that also will be useful when developing for VTEX).
+
+#### Node.js to run the tool
+- Install Node.js here: Toolbelt is written in JavaScript and Node.js. 
+Node is the engine that executes the JS and allows our tool to run and work at your computer) 
+- Install yarn here: yarn is a way to manage your code, organize content in packages 
+that you can download, share and... install, just like our CLI :) 
+
+Already has Node.js and Yarn? Lets install the Tooltbelt. 
+
+#### Installing the Toolbelt with yarn
+`yarn` powers app development to React, Node and VTEX IO! Install the Toolbelt using yarn and start working with us generating automatically all the files you'll need to create apps in the future. 
+To install it, run: 
+ 
+ ```
+ yarn global add vtex
+ ```
+
+<details>
+  <summary>Alternative installation using npm</summary>
+  You can install using `npm`. But, remember, if in the future 
+  you want develop an VTEX IO app, you'll need to install `yarn` 
```

Comment:
Not sure about one of the commas.

```suggestion
  You can install using `npm`. But remember, if in the future 
  you want to develop a VTEX IO app, you'll need to install `yarn` 
```
Should we specify? `if in the future you want to develop a React or Node.js app on VEX IO`

Reply:
Sure!!